entities/helper.py

```diff
@@ -0,0 +1,26 @@
+from typing import Any, Iterable
+
+from common import indent
+
+
+def indent_entity(entity: Any) -> str:
+    """indent an entity and add trailing newline
+    >>> indent_entity('(foo "1")')
+    ' (foo "1")\\n'
+    >>> indent_entity('(bar "2"\\n (baz "3")\\n)')
+    ' (bar "2"\\n  (baz "3")\\n )\\n'
+    """
+    result = '\n'.join(indent(1, str(entity).splitlines()))
+    result += '\n'
+    return result
+
+
+def indent_entities(entities: Iterable[Any]) -> str:
+    """indent a list of entities and add a trailing newline
+    >>> indent_entities(['(bar "2")', '(bar "3")'])
+    ' (bar "2")\\n (bar "3")\\n'
+    """
+    result = ''
+    for entity in entities:
+        result += indent_entity(entity)
```

Comment:
This isn't good style because strings are immutable.

Use a list instead:

```python
results = []
for entity in entities:
    results.append(indent_entity(entity))
return ''.join(results)
```

...or like this:

```python
return ''.join(indent_entity(entity) for entity in entities)
```

...or

```python
return ''.join(map(indent_entity, entities))
```

Reply:
Ohh much nicer, thanks :slightly_smiling_face: Fixed.